generator-honeycomb/app/templates/test/.eslintrc.yml

```diff
@@ -0,0 +1,4 @@
+globals:
+  browser: false
```

Comment:
See http://eslint.org/docs/user-guide/configuring#specifying-globals 🐱
I think you confuse `env` with `globals`.

Required for: https://github.com/danielbayerlein/honeycomb/blob/4e6317a63f7187ecd346f7ce855f18cf4d0e8757/generator-honeycomb/app/templates/test/integration/features/steps/index.js

We have already used it, see: https://github.com/danielbayerlein/honeycomb/blob/4e6317a63f7187ecd346f7ce855f18cf4d0e8757/generator-honeycomb/app/templates/test/_.eslintrc.yml#L2